services/web/client/source/class/osparc/desktop/preferences/pages/SecurityPage.js

```diff
@@ -122,33 +121,52 @@ qx.Class.define("osparc.desktop.preferences.pages.SecurityPage", {
       return form;
     },
 
-    __createValidTokenForm: function(service, key, secret) {
-      let form = new qx.ui.form.Form();
+    __createValidTokenForm: function(token) {
+      const service = token["service"];
```

Comment:
The tokens, once transmitted ... MUST NOT be stored anywhere on the client side

Reply:
I know, this PR is not closing the tokens issue, is just about not showing them.

In the same direction, when the frontend asks for the tokens, the backend should only answer with a list of service names.

Reply:
ok then...make sure to keep open then #1189